### PR TITLE
ISA updates for Cerebellin

### PR DESCRIPTION
**Pull request by djcomlab**
- Adds ISA creation tool `isa_create_metabo.xml`
- Updates ISA create tool, ISA-Tab validator, and isatab2json to produce and consume new `isa-tab` datatype
- Updates from @ilveroluca on mtblisa tools
- Updates container tags on containers that use new smaller isatools docker base image
- General cleanup.

### CODE REVIEW

File: config/phenomenal_tools2container.yaml [hunk 1]
```diff
@@ -76,15 +76,6 @@ assignment:
   max_pod_retrials: 3
   tools_id:
   - isa2w4m
-
```
Comment (pcm32) [inline config/phenomenal_tools2container.yaml]: Why is this going away?

File: config/phenomenal_tools2container.yaml [hunk 2]
```diff
@@ -394,13 +383,6 @@ assignment:
   max_pod_retrials: 3
   tools_id:
   - mzml2isa
-- docker_image_override: isajson-validator
```
Comment (pcm32) [inline config/phenomenal_tools2container.yaml]: This goes away as well?

Comment (pcm32) [reply config/phenomenal_tools2container.yaml]: Ahh, I see downwards that you are actually removing the wrapper... I thought this was a merging error... please neglect my previous comment...

File: config/phenomenal_tools2container.yaml (hunk 1 above)
Comment (pcm32) [reply config/phenomenal_tools2container.yaml]: Ok, I see downwards that you are deleting wrappers and other entries, so not a merging error that removed this.

**Comment by djcomlab** [comment]
@pcm32 Yes, testing in my local minikube. I have to admit there is likely to be another update needed from our side, but it would be great to get these changes merged in first.

**Comment by pcm32** [comment]
@djcomlab can you check the conflict resolution I made on job_conf? In particular, please check that none of the new tools are missing, and that all the tool name changes are coherent. I did my best! Please see https://github.com/phnmnl/container-galaxy-k8s-runtime/pull/123/commits/ee523c066959f95b035251fb12ca282e4bf9d1c0

After that, I'm happy to squash and merge.
Thanks!

**Comment by djcomlab** [comment]
The ISA-related tools I think look good to me in job_conf.xml

Looks like a bit of a monster merge!

**Comment by pcm32** [comment]
Yes, so much happened since you branched! Only conflict was in job_conf, didn't touch the rest.

**Pull request merged**